react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecText.java

```diff
@@ -23,11 +24,13 @@
 import com.facebook.react.views.textinput.ReactTextInputLocalData;
 import com.facebook.react.views.textinput.ScrollWatcher;
 
+import org.jetbrains.annotations.NotNull;
```

Comment:
Hmm, any particular reason for using the JetBrains `NotNull` instead of AndroidX's `NonNull`?

Asking since I'd find it kinda more "natural" to to use the AndroidX's annotations unless they don't suit us well.

Reply:
Left an additional comment on the particular change: https://github.com/wordpress-mobile/gutenberg-mobile/pull/1615/commits/a665093164086ad2f7ce6bed83e349da7cccfd46#r354960106